plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -295,4 +296,16 @@ public DeltaLakeConfig setCompressionCodec(HiveCompressionCodec compressionCodec
         this.compressionCodec = compressionCodec;
         return this;
     }
+
+    public boolean getRenameManagedTablesEnabled()
+    {
+        return renameManagedTablesEnabled;
+    }
+
+    @Config("delta.rename-managed-tables.enabled")
+    public DeltaLakeConfig setRenameManagedTablesEnabled(boolean renameManagedTablesEnabled)
```

Comment:
Add an `@ConfigDescription` for the setting for explaining the reasoning for this setting.